mri/tests/test_reconstructors.py

```diff
@@ -192,7 +192,7 @@ def test_self_calibrating_reconstruction(self):
             linear_op.n_coils = 1
             reconstructor = SelfCalibrationReconstructor(
                 fourier_op=fourier,
-                linear_op=linear_op,
+                linear_op=linear_opget_sample_data('3d-pmri'),
```

Comment:
this seems to be a typo

Reply:
wow, that will fail in tests, fixing. Thats surely a bad by-mistake paste by a scroll button press